Utils/src/CandidateTrackFilter.cc

```diff
@@ -0,0 +1,366 @@
+//-----------------------------------------------------------------------------------------
+//
+// Computation of the trackIsolation, for use with the isolated track veto 
+// used for the stop quark search in the single lepton channel
+// Author: Ben Hooberman
+//
+// For each PFCandidate above threshold minPt_PFCandidate store 4 quantities:
+// pT of PFCandidate
+// charge of PFCandidate
+// dz of PFCandidate w.r.t. the 1st good vertex
+// the trackIsolation value
+//
+// In the analysis, we veto any event containing IN ADDITION TO the selected lepton a charged PFCandidate with:
+// pT > 10 GeV, dz < 0.05 cm, and trackIso/pT < 0.1
+//
+//-----------------------------------------------------------------------------------------
+
+// system include files
+#include <memory>
+
+// CMSSW include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/global/EDFilter.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "DataFormats/Common/interface/ValueMap.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/Math/interface/deltaR.h"
+#include "DataFormats/Math/interface/deltaPhi.h"
+#include "DataFormats/Math/interface/Vector3D.h"
+#include "DataFormats/Math/interface/Point3D.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidateFwd.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+
+#include "TrackingTools/IPTools/interface/IPTools.h"
+#include "TrackingTools/TransientTrack/interface/TransientTrackBuilder.h"
+#include "TrackingTools/Records/interface/TransientTrackRecord.h"
+#include "MagneticField/Engine/interface/MagneticField.h"
+
+#include "CommonTools/UtilAlgos/interface/TFileService.h"
```

Comment:
unneeded include can be removed

Reply:
Done, if all you meant was the TFileService.h.